kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaServiceConfiguration.java

```diff
@@ -243,4 +243,10 @@
     )
     private Set<String> saslAllowedMechanisms = new HashSet<String>();
 
+    @FieldContext(
+            category = CATEGORY_KOP,
+            doc = "the number of reading entries from bookie"
```

Comment:
I think it's better to change the doc to `maximum number of entries that are read from cursor once per time` to note that the entries are read from cursors and the number is a **maximum** number as `ManagedCursor#readEntries`' doc says. The same goes with the name of `readEntriesNum`. Otherwise the config may be a little confused.

Reply:
`readEntriesNum -> maxReadEntriesNum` is ok?

Reply:
I think `maxReadCursorEntriesNum` is better